pom.xml

```diff
@@ -120,7 +121,7 @@
                     <rulesets>
                         <ruleset>/rulesets/java/braces.xml</ruleset>
                         <ruleset>/rulesets/java/strictexception.xml</ruleset>
-                        <ruleset>/rulesets/java/basic.xml</ruleset>
+                        <!--<ruleset>/rulesets/java/basic.xml</ruleset>-->
```

Comment:
Ok, then you need to look inside the target/site folder for the error and fix it. That's the correct way ! So please enable the plugin again, and fix the error. Thanks.